samples/snippets/src/main/java/com/example/spanner/TracingSample.java

```diff
@@ -58,7 +64,13 @@ public static void main(String[] args) throws Exception {
         .registerSpanNamesForCollection(Arrays.asList(SAMPLE_SPAN));
 
     // Installs an exporter for stack driver stats.
-    StackdriverStatsExporter.createAndRegister();
+    MonitoredResource.Builder builder = MonitoredResource.newBuilder();
```

Comment:
This is being added because without this an exception is thrown

```
Caused by: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Field timeSeries[0].resource.labels.project_id had an invalid value of "google.com:xxxxx: if present, must be the project number or ID in the request name.                                                      
        at io.grpc.Status.asRuntimeException(Status.java:533) 
        ```